src/transformers/htmlToPdf.test.ts

```diff
@@ -0,0 +1,112 @@
+import { execFile, ChildProcess } from 'child_process'
+import { mkdtemp, readFile, writeFile, unlink } from 'fs'
+import locateChrome from '../utils/locateChrome'
+import mockOf from '../../test/utils/mockOf'
+import htmlToPdf from './htmlToPdf'
+import fakeFile from '../../test/utils/fakeFile'
+
+const mockChromeLocation = 'chrome'
+
+jest.mock('chrome-location')
+jest.mock('child_process')
+jest.mock('fs')
+jest.mock('../utils/locateChrome')
+
+const execFileMock = mockOf(execFile)
+const mkdtempMock = mockOf(mkdtemp)
+const readFileMock = mockOf(readFile)
+const writeFileMock = mockOf(writeFile)
+const unlinkMock = mockOf(unlink)
+const locateChromeMock = mockOf(locateChrome)
+
+beforeEach(() => {
+  execFileMock.mockReset()
+  mkdtempMock.mockReset()
+  readFileMock.mockReset()
+  writeFileMock.mockReset()
+  unlinkMock.mockReset()
+  locateChromeMock.mockReset()
```

Comment:
Is there a way to make this happen automatically?

Reply:
Yes, [`resetMocks` config](https://jestjs.io/docs/en/configuration.html#resetmocks-boolean).